instrumentation/spring/spring-amqp-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/amqp/SpringAmqpMessageAttributesExtractor.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.amqp;
+
+import io.opentelemetry.instrumentation.api.instrumenter.messaging.MessageOperation;
+import io.opentelemetry.instrumentation.api.instrumenter.messaging.MessagingAttributesExtractor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.springframework.amqp.core.Message;
+
+final class SpringAmqpMessageAttributesExtractor
+    extends MessagingAttributesExtractor<Message, Void> {
+  @Override
+  protected String system(Message message) {
+    return "rabbitmq";
```

Comment:
maybe we should rename this module to `spring-amqp-rabbitmq`?

Reply:
Hmm, I'll rename it to `spring-rabbit` - this is the name of the library that actually contains `AbstractMessageListenerContainer`

Reply:
@mateuszrzeszutek I am still facing this issue. Trace Ids differ at RabbitListener annotated method. Btw I am using manual Otel instrumentation. Please guide me if I am missing anything obvious.